### PR TITLE
bug: Added missing out chain data

### PR DESCRIPTION
**Pull request by Cuteivist**
task: https://github.com/status-im/status-desktop/issues/12198

Status-desktop: https://github.com/status-im/status-desktop/pull/12705

* Passes data for second network for multi transactions

### CODE REVIEW

File: services/wallet/activity/details.go [hunk 1]
```diff
@@ -21,18 +21,23 @@ const (
...
-	GasLimit     hexutil.Uint64 `json:"gasLimit"`
-	TotalFees    *hexutil.Big   `json:"totalFees,omitempty"`
+	ID           string              `json:"id"`
+	MultiTxID    int                 `json:"multiTxId"`
+	Nonce        uint64              `json:"nonce"`
+	ChainDetails []EntryChainDetails `json:"chainDetails"`
+	Input        string              `json:"input"`
+	ProtocolType *ProtocolType       `json:"protocolType,omitempty"`
+	MaxFeePerGas *hexutil.Big        `json:"maxFeePerGas"`
+	GasLimit     hexutil.Uint64      `json:"gasLimit"`
```
Comment (stefandunca) [inline services/wallet/activity/details.go]: Why not `uint64`? What could `hexutil` specialization provide?

File: services/wallet/activity/details.go [hunk 2]
```diff
@@ -56,14 +61,21 @@ func getMultiTxDetails(ctx context.Context, db *sql.DB, multiTxID int) (*EntryDe
...
 		tx_hash,
 		blk_number,
+		network_id,
 		type,
 		account_nonce,
-		tx,
-		contract_address
+		contract_address,
+		CASE 
+			WHEN json_extract(tx, '$.gas') = '0x0' THEN NULL
```
Comment (stefandunca) [inline services/wallet/activity/details.go:75]: Did we upgrade sqlite to have access to `json` functionality?

**Review by stefandunca — APPROVED** [review]
🍯

File: services/wallet/activity/details.go (hunk 2 above)
Comment (Cuteivist) [reply services/wallet/activity/details.go:75]: We are already using it in activity filter sql for some time.

File: services/wallet/activity/details.go (hunk 1 above)
Comment (Cuteivist) [reply services/wallet/activity/details.go]: Good point. Updated to uint64. Tested, it works.

**Pull request merged**